lib/mayaHydra/hydraExtensions/sceneIndex/mayaHydraSceneIndexUtils.h

```diff
@@ -24,8 +24,10 @@
 #include <pxr/imaging/hd/materialNetworkSchema.h>
 #include <pxr/imaging/hd/materialNodeSchema.h>
 #include <pxr/imaging/hd/materialSchema.h>
+#include <pxr/imaging/hd/perfLog.h>//For HD_TRACE_FUNCTION
```

Comment:
This seems unused

Reply:
It's actually used for HD_TRACE_FUNCTION, removing it doesn't compile on different platforms.